src/Mvc/Mvc.ViewFeatures/test/RazorComponents/HtmlRendererTest.cs

```diff
@@ -316,6 +316,28 @@ public void RenderComponentAsync_MarksSelectedOptionsAsSelected()
         AssertHtmlContentEquals(expectedHtml, result);
     }
 
+    [Fact]
+    public void RenderComponentAsync_RendersValueAttributeAsTextContentOfTextareaElement()
+    {
+        // Arrange
+        var expectedHtml = "<textarea rows=\"10\" cols=\"20\">Hello World!</textarea>";
+        var serviceProvider = new ServiceCollection().AddSingleton(new RenderFragment(rtb =>
+        {
+            rtb.OpenElement(0, "textarea");
+            rtb.AddAttribute(1, "value", "Hello World!");
```

Comment:
Would it be possible to change this to something with angle-brackets in just to show the output is correctly encoded? For example:

```suggestion
            rtb.AddAttribute(1, "value", "Hello <html>-encoded content!");
```

... with the assertion being changed to look for `Hello &lt;html&gt;-encoded content!` or similar.

Reply:
thanks! done